src/django/api/views/boundary.py

```diff
@@ -52,6 +56,39 @@ def get_requested_utility_ids(request):
         return None
 
 
+class BoundarySubmitView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def patch(self, request, id, format=None):
+        if request.user.role not in [Roles.CONTRIBUTOR, Roles.ADMINISTRATOR]:
+            raise ForbiddenException(
+                "Only contributors and administrators can submit boundaries."
+            )
+
+        boundary_set = get_boundary_queryset_for_user(request.user)
+        boundary_set = boundary_set.prefetch_related("submissions")
+        boundary = get_object_or_404(boundary_set, pk=id)
+        if boundary.status != BOUNDARY_STATUS.DRAFT:
+            raise BadRequestException(
+                "Cannot submit boundary with status: {}".format(boundary.status.value),
+            )
+
+        now = datetime.now(tz=timezone(settings.TIME_ZONE))
+        yyyy_mm_dd = now.isoformat()[:10]
+        fp = f"{boundary.utility.pwsid}_{boundary.utility.address_city}_{yyyy_mm_dd}"
```

Comment:
This might change depending on 991a6a61b04c6b5062c29cd8add44381fea970fc. In which case I would be able to download the map in this PR after all (once it lands).

Reply:
Probably better to wait until the file storage is firmed up.